tests/appsec/test_automated_login_events.py

```diff
@@ -155,7 +165,14 @@ def test_login_sdk_failure(self):
                 assert meta["appsec.events.users.login.failure.track"] == "true"
                 assert meta["appsec.events.users.login.failure.usr.id"] == "sdkUser"
                 assert meta["appsec.events.users.login.failure.usr.exists"] == "true"
-                assert meta["manual.keep"] == "true"
+                self.assert_priority(span, meta)
+
+    def assert_priority(self, span, meta):
+        assert (
```

Comment:
for better debugging, I would rewrite this as : 

```python
if span["metrics"]["_sampling_priority_v1"] != self.MANUAL_KEEP:
    assert "manual.keep" in meta, "manual.keep should be in meta when _sampling_priority_v1 is not MANUAL_KEEP"
    assert meta["manual.keep"] == "true", 'meta.manual.keep should be "true" when _sampling_priority_v1 is not MANUAL_KEEP'
```

Reply:
changed in https://github.com/DataDog/system-tests/pull/1347/commits/c4d6c8c814f7842d98dddeab2638c8927bed378e